### ISSUE TITLE
What project template is used to kick start cppwinrt?

### ISSUE THREAD

**Issue by Noemata** [Microsoft/cppwinrt#10]
What project template if any do you use to kick start a C++ UWP solution using cppwinrt (Blank C++ UWP APP)? It would also be helpful to show real XAML integration. A simple multi-page XAML app would be a good start. A redo of the RSS reader sample would certainly cover a lot of ground.

Lastly, showing a sample that integrates cppwinrt with C#/XAML solution would be most welcome; two variants, C# as the GUI and cppwinrt providing a component service, and cppwinrt as the GUI and C# providing the component service.

**Comment by kennykerr** [comment]
I would suggest you start with a copy of one of [these Visual C++ projects](https://github.com/Microsoft/cppwinrt/tree/master/10.0.14393.0/Samples) as your starting point. It has everything already pre-configured to work with C++/WinRT. 

Support for Xaml and components will be available in a future update.

**Comment by Noemata** [comment]
Thanks, that was what I was doing, using a copy and tweaking.  I looked through all the samples, much nicer code. I'm not a C++ purist but I did find the CLI like syntax of C++/CX superfluous given how effectively C++ can be morphed and prodded (and letting the compiler do the heavy lifting).  Looking forward to seeing component examples and whether you'll come up with a way of hiding MIDL.  Hopefully the rest of Microsoft will accept that this is the right path forward.
